SUIApp/SUIApp/TabBarView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ContentView.swift
+//  SUIApp
+//
+//  Created by Vladka on 16.01.2023.
+//
+
+import SwiftUI
+
+/// Таббар
+struct TabBarView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let vpnTitleText = "VPN"
+        static let rateTitleText = "Rate"
+        static let basketTitleText = "Basket"
+        static let vpnSystemImageNameText = "lock"
+        static let rateSystemImageNameText = "dollarsign"
+        static let basketSystemImageNameText = "basket"
+        static let zeroNumber = 0
+        static let oneNumber = 1
+        static let twoNumber = 2
+    }
+ 
+    var body: some View {
```

Comment:
```suggestion
    // MARK: - Body

    var body: some View {
```
Кидали скрин в важное, как я понял body маскируется отдельно, но могу ошибаться